src/k2/cmd/tpcc/transactions.h

```diff
@@ -241,8 +241,72 @@ class PaymentT : public TPCCTxn
         return writeRow<History>(history, _txn).discard_result();
     }
 
+    future<> getCIdByLastNameViaIndex() {
```

Comment:
The order status transaction also selects customer by last name and can use the secondary index.

Reply:
replaced getCustomerId() with getCustomerIdViaIndex() for the order status transaction.